src/analysis/normed/normed_field.lean

```diff
@@ -57,6 +57,18 @@ class normed_ring (α : Type*) extends has_norm α, ring α, metric_space α :=
 (dist_eq : ∀ x y, dist x y = norm (x - y))
 (norm_mul : ∀ a b, norm (a * b) ≤ norm a * norm b)
 
+/-- A normed division ring is a division ring endowed with a seminorm which satisfies the inequality
+`∥x y∥ = ∥x∥ ∥y∥`. -/
+class normed_division_ring (α : Type*) extends has_norm α, division_ring α, metric_space α :=
+(dist_eq : ∀ x y, dist x y = norm (x - y))
+(norm_mul' : ∀ a b, norm (a * b) = norm a * norm b)
```

Comment:
Very minor, but pick a side in the letters war.

Reply:
This is like this throughout the file, who knows why. I didn't change it here or elsewhere only to minimize the diff.